Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG PROJ_VERSION="6.3.0"
+
+FROM rust:1-slim-stretch as builder
+ARG PROJ_VERSION
+ENV PROJ_DEB "proj_${PROJ_VERSION}_amd64.deb"
+ENV GPG_KEY "C60D758F807A525534C5DFD57B639E3638A8112A"
+WORKDIR /usr/src/app
+RUN apt update && apt install --yes apt-transport-https gnupg2 wget
+RUN wget --quiet --output-document - "https://kisiodigital.jfrog.io/kisiodigital/api/gpg/key/public" | apt-key add -
+RUN echo "deb [arch=amd64] https://kisiodigital.jfrog.io/kisiodigital/debian-local stretch main" > /etc/apt/sources.list.d/kisio-digital.list
+RUN apt update && apt install --yes pkg-config libssl-dev clang proj=${PROJ_VERSION}
+
+COPY gtfs2netexfr gtfs2netexfr
+COPY gtfs2ntfs gtfs2ntfs
+COPY ntfs2gtfs ntfs2gtfs
+COPY ntfs2netexfr ntfs2netexfr
+COPY ntfs2ntfs ntfs2ntfs
+COPY Cargo.toml .
+COPY src src
```

Comment:
Why not `COPY . .`? I think because you defined the `.dockerignore`, it will copy only what's not ignored (see https://docs.docker.com/engine/reference/builder/#dockerignore-file).

Reply:
Ok. Besides, it was done like this previously.

Reply:
Yes, I know, incremental improvements ;-) Whoever did that previously was improving something else at the time I guess!